ddtrace/contrib/trace_utils.py

```diff
@@ -106,12 +107,46 @@ def ext_service(pin, config, default=None):
     return default
 
 
-def set_http_meta(span, config, method=None, url=None, status_code=None):
+def get_error_codes():
```

Comment:
A docstring here would be helpful to get an idea of what is happening

Reply:
Also a cleaner approach might be to have a function `is_error_code()` which simply returns whether a code is erroneous or not, rather than having to construct a list of all error ranges which then has to be checked.